### PR TITLE
Adapt examples for SDL version 2.1.1

### PR DESCRIPTION
**Pull request by zzeekk**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -32,7 +32,7 @@
 		  If version cannot be resolved, make sure maven central repository is defined in settings.xml and the corresponding profile activated.
 		  If version in IntelliJ still cannot be resolved, a restart of IntelliJ might help!
 		-->
-		<version>2.0.3</version>
+		<version>2.1.1-SNAPSHOT</version>
```
Comment (Geheiner) [inline pom.xml:35]: did you replace a release version with a snapshot version on purpose?

**Review by pgruetter — APPROVED** [review]
LGTM, fixed the build error

**Review by Geheiner — CHANGES_REQUESTED** [review]
it looks good (I just have that one comment)

File: pom.xml (hunk 1 above)
Comment (zzeekk) [reply pom.xml:35]: To be discussed in the SDL weekly. For now this PR shows you the changes to the latest version.

**Comment by zzeekk** [comment]
> LGTM, fixed the build error
Thanks @pgruetter, definitely an advantage to have SNAPSHOT builds :-)

**Review by Geheiner — APPROVED** [review]

**Pull request merged**